addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -149,6 +150,15 @@ export class ListCorePlugin extends OdooCorePlugin {
                 );
                 break;
             }
+            case "UPDATE_ODOO_LIST": {
```

Comment:
Each time we add a new core command, we should think about the transformations with all others core commands. Here are some links to the documentation which explain that process:
- [Transformations](https://github.com/odoo/o-spreadsheet/blob/saas-17.3/doc/integrating/collaborative/collaborative.md#transformations)
- [OT](https://github.com/odoo/o-spreadsheet/blob/saas-17.3/doc/integrating/collaborative/collaborative.md#operational-transform)
- [History](https://github.com/odoo/o-spreadsheet/blob/saas-17.3/doc/integrating/collaborative/collaborative.md#history)

You can find exemples [here](https://github.com/odoo/enterprise/blob/master/spreadsheet_edition/static/src/bundle/list/operational_transform.js#L17)